### DISCUSSION TITLE
Question on settings API release timeline

### DISCUSSION THREAD

**Discussion by mkvani** [ansys/pyfluent#458] [Q&A] [answered]
Hi All,

We are using PyFluent for one of the solutions project. We aim to release the solution by 23R1 release. 
As TUI API do not provide useful exception handling response, we are using settings APIs. 

Settings API is currently a beta feature. Could you please provide information on when we plan to make it a release feature?

Thank you,
Manoj

**Comment by dnwillia-work** [comment, marked as answer]
Hi Manoj, we have been discussing this with dev/testing and doc teams in the past couple weeks.  We would like to get it released at 2023 R1 for 'pro level' physics, but there are some risks that might mean keeping it beta for another release.

Regarding the TUI and exception handling this should be working, there is an open issue to look into that.

**Reply by mkvani** [reply]
Thank you @dnwillia-work for the information. This is helpful.

About TUI exception handling, could you please provide information about the open issue?

Best regards,
Manoj

**Reply by dnwillia-work** [reply]
I meant issue #392 is there to track this topic, which you seem to be aware of already.  Reading Mainak's response I guess there is no uniform way to guarantee that the TUI commands always throw exceptions.  So, this would be a feature requirement on Fluent rather than the API.
